etna/models/tbats.py

```diff
@@ -83,6 +86,124 @@ def get_model(self) -> Model:
         """
         return self._fitted_model
 
+    def forecast_components(self, horizon: int = 1) -> pd.DataFrame:
+        """Estimate forecast components.
+
+        Parameters
+        ----------
+        horizon:
+            forecast horizon
+
+        Returns
+        -------
+        :
+            dataframe with forecast components
+        """
+        self._check_components()
+        raw_components = self._decompose_forecast(horizon=horizon)
+        components = self._named_components(raw_components=raw_components)
+
+        return pd.DataFrame(data=components)
+
+    def predict_components(self, horizon: int = 1) -> pd.DataFrame:
+        """Estimate prediction components.
+
+        Parameters
+        ----------
+        horizon:
+            forecast horizon
+
+        Returns
+        -------
+        :
+            dataframe with prediction components
+        """
+        raise NotImplementedError("Prediction decomposition isn't currently implemented!")
+
+    def _check_components(self):
+        """Compare fitted model params with the initial params."""
```

Comment:
May you add comments about why we need this check in docstring -- some components might be not fitted because .....